mods/d2k/rules/structures.yaml

```diff
@@ -59,8 +60,9 @@ construction_yard:
 	RevealsShroud:
 		Range: 10c0
 	Production:
-		Produces: Building, Upgrade
+		Produces: Building
 	Exit:
```

Comment:
I don't think we need this anymore, so this trait can be removed (it was only used for upgrades, which have now moved to the player).